docs/tutorials/mac_experimental.md

```diff
@@ -0,0 +1,99 @@
+# Using podman-machine on MacOS (x86_64 and Apple silicon)
+
+## SetUp
+
+You must obtain a compressed tarball that contains the following:
```

Comment:
Probably a comment for later but we should let them know where to get the tarball from in step 8.

Reply:
well of course, but rn, i have no clue